docs/visualize/controls.asciidoc

```diff
@@ -0,0 +1,93 @@
+[[controls]]
+== Controls Visualization
+
+The Controls visualization enables you to add interactive inputs
+to Kibana dashboards.  You can create two types of inputs:
+a dropdown menu and a radio slider.
+
+To start a *Controls* visualization, open the Visualization application
+and click the *+* button. Scroll to the *Others* section and
+select *Controls*.
+
+[role="screenshot"]
+image::images/controls/controls_in_dashboard.png[]
+
+[[add-input-controls]]
+=== Adding Input Controls
+
+From the *Controls* tab, you can add a dropdown menu
+and range slider to your visualization.
+
+==== Dropdown menu
+
+A dropdown menu allows users to filter content by selecting
+one or more options from a list. The dropdown menu is dynamically populated
+with the results of a terms aggregation.
+
+[role="screenshot"]
+image::images/controls/dropdown_control_editor.png[]
+
+*Control Label*:: The label for the dropdown menu. By default, the
+label is the field name.
+
+*Index Pattern*:: The <<index-patterns,index pattern>> that contains
+the data set to visualize.
+
+*Field*:: The field to use to populate the options list
+and to filter on when users interact with the input.
+The list of available fields is derived from the specified
+index pattern.
+
+*Parent control*:: The control for chaining dropdown menus so that the
+selection in the first menu
+filters the terms in the second menu. Only available when
+creating multiple dropdown menus.
+
+*Multiselect*:: When enabled, the dropdown menu allows users to select multiple options.
+
+*Size*:: The number of options to include in the list.
+
+==== Range slider
+
+A range sliders allow users to filter content within a range of numbers.
+The range slider minimum and maximum values are dynamically populated with
+the results of a min and max aggregation.
+
+[role="screenshot"]
+image::images/controls/range_slider_editor.png[]
+
+*Control Label*:: The label for the range slider. By default, the
+label is the field name.
+
+*Index Pattern*:: The <<index-patterns,index pattern>> that contains
+the data set to visualize.
+
+*Field*:: The field to use to populate the range slider
+and to filter on when users interact with the input.
+The list of available fields is derived from the
+specified index pattern.
+
+*Step Size*:: The increment/decrement size of the slider.
+
+*Decimal Places*:: The number of decimal places.
+
+[[configure-input-control-options]]
+=== Configuring Settings for all Inputs
```

Comment:
I still like calling this section "Options". "Configuring Settings for all Inputs" is complicated and a little misleading. The settings only affect a single Controls visualization and are not global settings. I think the sentence below explains that better than a title block